benchmarks/run.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/sh
```

Comment:
Whoops, I missed this during my review, but this should be `#!/bin/sh`.

Reply:
Fixed, thanks.